### PR TITLE
Rename Token from OVL to OV

### PR DESCRIPTION
**Pull request by magnetto90**
Update token symbol and variable names

### CODE REVIEW

File: contracts/OverlayV1Token.sol [hunk 1]
```diff
@@ -2,30 +2,20 @@
 pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
```
Comment (TomasCImach) [inline contracts/OverlayV1Token.sol:5]: Propose this change in a different PR.

File: tests/OverlayV1Token.t.sol
Comment (TomasCImach) [inline tests/OverlayV1Token.t.sol:1]: Migrate to Refactor PR

**Review by TomasCImach — CHANGES_REQUESTED** [review]
Renaming looks good. Migrate Refactor of AccessControl to other PR.

File: tests/OverlayV1Token.t.sol
Comment (magnetto90) [reply tests/OverlayV1Token.t.sol:1]: Done in #141 but this one has to be merge before.

**Review by TomasCImach — APPROVED** [review]
Looks good! Tests are passing

**Review by EperezOk — APPROVED** [review]

**Pull request merged**